pkgs/development/embedded/tytools/default.nix

```
@@ -5,34 +5,58 @@
   cmake,
   pkg-config,
   wrapQtAppsHook,
-  qtbase,
+  qt6,
 }:
 
 stdenv.mkDerivation rec {
   pname = "tytools";
-  version = "0.9.8";
+  version = "0.9.9";
 
   src = fetchFromGitHub {
     owner = "Koromix";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-MKhh0ooDZI1Ks8vVuPRiHhpOqStetGaAhE2ulvBstxA=";
+    repo = "rygel";
+    tag = "tytools/${version}";
+    sha256 = "sha256-nQZaNYOTkx79UC0RHencKIQFSYUnQ9resdmmWTmgQxA=";
```

Optional:

```suggestion
    hash = "sha256-nQZaNYOTkx79UC0RHencKIQFSYUnQ9resdmmWTmgQxA=";
```

Disclaimer: Yeah I know that it should be done automatically in `nixpkgs`, but since we don't have the tool to do it and that this line has been modified, as a maintainer, I have to notify it. And this is optional, so, it's a nice to have, but not blocking for proceeding further.